src/types/initial-data.ts

```diff
@@ -0,0 +1,8 @@
+export interface IInitialData {
```

Comment:
I would recommend to define this specifically. So we have a reference on what exists on the initialData.

Reply:
I did nod change this because the reason for that is just to define `window.initialData` globally but i added specific interfaces `IInitialStoreDate` in the stores where the content of the initial store data is described.